src/cmd/create.go

```diff
@@ -699,10 +700,20 @@ func pullImage(image, release, authFile string) (bool, error) {
 	}
 
 	if promptForDownload {
-		fmt.Println("Image required to create toolbox container.")
+		infoSize, err := skopeo.Inspect(imageFull)
+		if err != nil {
+			logrus.Debugf("Getting image size failed: %s", err)
+
+			fmt.Println("Image required to create toolbox container.")
 
-		prompt := fmt.Sprintf("Download %s (500MB)? [y/N]:", imageFull)
-		shouldPullImage = askForConfirmation(prompt)
+			prompt := fmt.Sprintf("Download %s (500MB)? [y/N]:", imageFull)
+			shouldPullImage = askForConfirmation(prompt)
+		} else {
+			fmt.Println("Image required to create toolbox container.")
+
+			prompt := fmt.Sprintf("Download %s (%sMB)? [y/N]:", imageFull, infoSize)
```

Comment:
The `MB` is getting duplicated because it's part of the human-readable size.  :)